src/componentprocessor/index.ts

```diff
@@ -82,8 +82,12 @@ export class ComponentDefinition {
     }
 }
 
-export async function getComponentDefinitionMap(rootDir: string) {
-    const componentsPattern = rootDir + "/components/**/*.xml";
+export async function getComponentDefinitionMap(
+    rootDir: string = "",
+    additionalDirs: string[] = []
+) {
+    let searchString = `{components,${additionalDirs.join(",")}}`;
```

Comment:
I assume the trailing comma doesn't cause issues with `fastglob`?

Reply:
It does not! In fact, if you _don't_ have a trailing comma with one-item list, then it _does_ cause issues with `fastglob`, strangely enough.